src/main/java/com/bio4j/model/uniprot/programs/ImportUniProtEdges.java

```diff
@@ -761,19 +761,19 @@ private void importProteinGeneNameEdges(
   {
 
     Optional.ofNullable(entryXMLElem.asJDomElement().getChild(GENE_TAG_NAME))
-    .ifPresent(
-    geneElement -> geneElement.getChildren(GENE_NAME_TAG_NAME)
-    .stream()
-    .forEach(
-    geneNameElem -> {
-      graph.geneNameNameIndex().getVertex(geneNameElem.getText()).ifPresent(
-      geneName ->
-      protein.addOutEdge(graph.ProteinGeneName(), geneName)
-      .set(graph.ProteinGeneName().geneNameType, geneNameElem.getAttributeValue("type"))
+      .ifPresent(
+        geneElement -> geneElement.getChildren(GENE_NAME_TAG_NAME)
+          .stream()
```

Comment:
This `.stream()` application is a bit confusing. How about this way of formatting?

```java
  ...
  .ifPresent( geneElement -> 
    geneElement.getChildren(GENE_NAME_TAG_NAME)
     .stream()
     .foreach( geneNameElem -> {
       ...
     }
  )
```